v1/templates/nodered-mininal/README.md

```diff
@@ -0,0 +1,9 @@
+# nodered
+NodeRed Docker Template
```

Comment:
Could you add a short description of nodered, and link to https://nodered.org/

Reply:
done.